### ISSUE TITLE
Hold Movies until they finish

### ISSUE THREAD

**Issue by raspberry-soft** [renpy/renpy#5002]
Implement a way to let the movie ends before showing the next displayable?

Quick use case and example:

```
show animated eileen dance step one  # movie
with dissolve
e "First movement"

show animated eileen dance step two # movie
with dissolve
e "Second movement"

show cg eileen resting  # image
with dissolve
e "I'm done."

```
In this example, the eileen dance is animated and looped, and the next animation starts exactly after the last frame of the previous animation. The same with the still image.

Is there a way to let the video or loop to play until the end even after the player clicks? Because if it suddenly changes or stops,  it  looks wonky. Think of it like Live2D movements transitions between. It seems like it's alive and not glitched.

Something similar can be achieved if both animations are in the same channel. It will wait until the first is finished to play the next. But if the video is stopped by the next statement the video suddenly changes into the still image and it looks chopped.
Also, if they are in the same channel, the transition plays when the player clicks and not when the next movie plays (as it's delayed until the previous finishes)

TLDR
Opt-in holding the movie until it finishes when using the `show` statement.
Using the `scene` statement will end it suddenly.

**Comment by MiiNiPaa** [comment]
There is a way to do that, though it is kinda obscure and requires carefully reading the documentation (emphasis mine):

> `channel`

>    The audio channel associated with this movie. **When a movie file is played on that channel, it will be displayed in this Movie displayable.** If this is left at the default of "movie", and play is provided, a channel name is automatically selected, using [config.single_movie_channel](https://www.renpy.org/doc/html/config.html#var-config.single_movie_channel) and [config.auto_movie_channel](https://www.renpy.org/doc/html/config.html#var-config.auto_movie_channel).

You can manupulate movies playing on that channel the same way you can manipulate music playing. Including replacing it, queuing it, etc.

This is an example code:
```
﻿image m1 = Movie(channel='bg_movies', play="nothing.webm", loop=True)

label start:
    show m1
    "Nothing is shown here"
    $ renpy.music.play("letters.webm", channel='bg_movies')
    "letters are flying"
    $ renpy.music.queue("earth.webm", channel='bg_movies')
    "Animation of floating letters runs until the end and then is replaced by animation of Earth rotating"

    return

```

**Comment by renpytom** [comment]
This doesn't seem like something I'd want to implement. When the user clicks, they're asking to advance the game, and denying them that isn't something the engine should be doing.
